TESTS/mbed_hal/qspi/flash_configs/MT25Q_config.h

```diff
@@ -0,0 +1,249 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018-2018 ARM Limited
+ *
```

Comment:
can you add SPDX identifier also for new files in HAL (HAL implementation or tests in this case) please

Reply:
I have made this change